amqp_influxdb/__main__.py

```diff
@@ -20,13 +20,21 @@
     InfluxDBPublisher,
     AMQPTopicConsumer,
     BATCH_SIZE,
-    MAX_BATCH_DELAY)
+    MAX_BATCH_DELAY,
+    BROKER_PORT_SSL,
+    BROKER_PORT_NO_SSL,
+)
 
 
 def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument('--amqp-hostname', required=False,
                         default='localhost')
+    parser.add_argument('--amqp-username', required=True)
```

Comment:
why are usename and password required. This is not consistent with RabbitMQ ecosystem in which guest:guest defaults are assumed unless stated otherwise